esp/services/ws_store/espstorelib/daliKVStore.cpp

```diff
@@ -410,14 +410,19 @@ bool CDALIKVStore::fetch(const char * storename, const char * ns, const char * k
     {
         xpath.appendf("/%s", key);
         if(!storetree->hasProp(xpath.str()))
-            throw makeStringExceptionV(MSGAUD_user, -1, "DALI Keystore fetch: invalid key '%s' detected!", key);
-
-        value.set(storetree->queryProp(xpath.str()));
+        {
+            LOG(MCuserInfo,"DALI Keystore fetch: invalid key '%s' detected!", key);
+            value.clear();
+        }
+        else
+        {
+            value.set(storetree->queryProp(xpath.str()));
+        }
 
         return value.str();
     }
     else
-        throw MakeStringException(-1, "DALI Keystore fetch: Namespace not provided!");
+        throw MakeStringException(-1, "DALI Keystore fetch: Key not provided!");
```

Comment:
Minor: change the MakeStringException() to makeStringException() since you are changing this code line.